### ISSUE TITLE
Uncaught Error: Invariant Violation: findComponentRoot

### ISSUE THREAD

**Issue by jpatel3** [seatgeek/react-slider#12]
Where there more than one slider on the page, it throws following error upon updating the subsequent sliders - 

Uncaught Error: Invariant Violation: findComponentRoot(..., .2.2.0.1.0.0): Unable to find element. This probably means the DOM was unexpectedly mutated (e.g., by the browser), usually due to forgetting a <tbody> when using tables, nesting tags like <form>, <p>, or <a>, or using non-SVG elements in an <svg> parent. Try inspecting the child nodes of the element with React ID ``.

It may be some configuration issue on my code, any help/pointers would be helpful.

**Comment by jpatel3** [comment]
I think the issue is with decimal point range. Its not supported currently, however the above error is not accurate.

**Comment by nishvs** [comment]
Is there any working sample for this component?

**Comment by jpatel3** [comment]
No. They are not currently not adding any more effort to this component. I tried to implement it but there are various issues with the component.
